eland/tests/dataframe/test_sample_pytest.py

```diff
@@ -0,0 +1,103 @@
+# Licensed to Elasticsearch B.V under one or more agreements.
+# Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+# See the LICENSE file in the project root for more information
+
+# Copyright 2020 Elasticsearch BV
```

Comment:
This file looks like it has a duplicate license header, can you remove the longer one and only leave the 3 line header :)